### PR TITLE
fix: multiples clicks prevented

### PR DESCRIPTION
**Pull request by Clearwood**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

This PR prevents multiple clicks to result in the function being called multiple times.

### CODE REVIEW

**Comment by Clearwood** [comment]
> how to reproduce the issue?

To reproduce you need to click on a the video more than once before the popup appears which is possible as the click event handler is not deactivated

File: packages/streaming-site-example/js/app.ts [hunk 1]
```diff
@@ -36,11 +37,14 @@ new Vue({
...
     methods: {
       getPopup: function (video) {
+        if(this.popup.on) return;
+        this.iframe = true;
         this.initiateWallet(video.tokenId);
         this.popup.video = video;
         this.popup.on = true;
       },
       removePopup: function () {
+        if (this.iframe) return;
```
Comment (troggy) [inline packages/streaming-site-example/js/app.ts:47]: `removePopup` is used to hide the video, when user clicks away. I guess we need to hide the iframe as well in this situation

Comment (Clearwood) [reply packages/streaming-site-example/js/app.ts:47]: Yes we can either hide the iframe in that circumstance or don't allow the user to click away. I would argue for the second option as it makes the user focus on the iframe and having to acknowledge it.

Comment (troggy) [reply packages/streaming-site-example/js/app.ts:47]: let's remove the "click outside the movie" then? We will need another way to close the popup then

Comment (Clearwood) [reply packages/streaming-site-example/js/app.ts:47]: What I am doing is blocking the closing of the popup until the iFrame is closed, in general I feel the idea of clicking outside the movie is intuitive

Comment (troggy) [reply packages/streaming-site-example/js/app.ts:47]: ah, now I get it!

**Pull request merged**